mx.sulong/mx_sulong.py

```diff
@@ -419,13 +426,30 @@ def runTypeTestCases(args=None):
     vmArgs, _ = truffle_extract_VM_args(args)
     return unittest(getCommonUnitTestOptions() + vmArgs + ['com.oracle.truffle.llvm.types.floating.test'])
 
-def getCommonOptions():
-    return ['-Dgraal.TruffleCompilationExceptionsArePrinted=true', '-Dgraal.ExitVMOnException=true', '-Dsulong.IntrinsifyCFunctions=false', getSearchPathOption()]
+default_lib_args = ['-lgmp', '-lgfortran']
+
+def getCommonOptions(lib_args=default_lib_args):
```

Comment:
I think that the [suOptBench](https://github.com/chrisseaton/sulong/blob/library-options/mx.sulong/mx_sulong.py#L542) benchmark runner should link all the libraries by default to make the automatic execution of the benchmarks easier. I can also address this in a subsequent PR.